src/main/java/de/tum/in/www1/artemis/service/ParticipationService.java

```diff
@@ -935,6 +1001,22 @@ public void deleteAllByExerciseId(Long exerciseId, boolean deleteBuildPlan, bool
         }
     }
 
+    /**
+     * Delete all participations belonging to the given team
+     *
+     * @param teamId the id of the team
+     * @param deleteBuildPlan specify if build plan should be deleted
+     * @param deleteRepository specify if repository should be deleted
+     */
+    @Transactional
+    public void deleteAllByTeamId(Long teamId, boolean deleteBuildPlan, boolean deleteRepository) {
+        List<StudentParticipation> participationsToDelete = findByTeamId(teamId);
```

Comment:
please add a log.info(..) statement here so that we could potentially track this in the log

Reply:
Added